apps/meteor/app/models/server/raw/Sessions.ts

```diff
@@ -737,12 +739,333 @@ export const aggregates = {
 };
 
 export class SessionsRaw extends BaseRaw<ISession> {
+	protected indexes: IndexSpecification[] = [
+		{ key: { instanceId: 1, sessionId: 1, year: 1, month: 1, day: 1 } },
+		{ key: { instanceId: 1, sessionId: 1, userId: 1 } },
+		{ key: { instanceId: 1, sessionId: 1 } },
+		{ key: { sessionId: 1 } },
+		{ key: { userId: 1 } },
+		{ key: { userId: 1, sessionId: 1 } },
+		{ key: { userId: 1, loginToken: 1, logoutAt: 1 } },
+		{ key: { loginToken: 1, logoutAt: 1 } },
+		{ key: { loginToken: 1 } },
+		{ key: { year: 1, month: 1, day: 1, type: 1 } },
+		{ key: { type: 1 } },
+		{ key: { ip: 1, loginAt: 1 } },
+		{ key: { createdAt: 1 } },
+		{ key: { _computedAt: 1 }, expireAfterSeconds: 60 * 60 * 24 * 45 },
+		{
+			key: {
+				'device.name': 'text',
+				'device.type': 'text',
+				'device.os.name': 'text',
+				'sessionId': 'text',
+				'userId': 'text',
+			},
+			weights: {
+				'device.name': 10,
+				'device.os.name': 10,
+				'device.type': 5,
+				'sessionId': 1,
+				'userId': 1,
+			},
+			name: 'text_search_w_weights',
+		},
+	];
+
 	private secondaryCollection: Collection<ISession>;
 
 	constructor(public readonly col: Collection<ISession>, public readonly colSecondary: Collection<ISession>, trash?: Collection<ISession>) {
 		super(col, trash);
 
 		this.secondaryCollection = colSecondary;
+		this.col.createIndexes(this.indexes);
```

Comment:
Isn't this done on `BaseRaw`? 👀

Reply:
Just works for me when I add this line. But, I need to add this before this [patch release](#25101). 
I skip it, I'll remove it!

Reply:
@sampaiodiego , were you aware of this? 👀 like, index creation is not working on baseraw

Reply:
It was fixed one or two releases ago.

Reply:
I quite remember that, but was not sure, thanks Pierre!

So, @albuquerquefabio , you can trust on BaseRaw to create indexes now 👀